pkgs/development/python-modules/ffmpeg-python/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, future
+}:
+
+buildPythonPackage rec {
+  pname = "ffmpeg-python";
+  version = "0.2.0";
+
+  src = fetchPypi {
+    sha256 = "09s19kvgc4f0g05z3xjj8w15xcwbabmv3j0i1vppii978srms8k5";
+    inherit pname version;
+  };
+
+  propagatedBuildInputs = [ future ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Python bindings for FFmpeg - with complex filtering support";
```

Comment:
shouldn't this be built against ffmpeg?

Reply:
Well yes, if you mean it should provide `ffmpeg` as a runtime dependency. I did a force push to add `ffmpeg` to `propagatedUserEnvPkgs`. This should fix it up. Thank you for having a look.

Reply:
does it shell out to the cli utility? Or does it link against a shared library, they have very different implications :)

Reply:
It builds a command line and then calls the ffmpeg cli utility.

Reply:
You'll want to patch how it calls the command, so that it calls it directly like
```
   popen.call("ffmpeg"....)
```
to
```
  popen.call("${ffmpeg}/bin/ffmpeg"....)
```

Reply:
Thanks for the input, it should all be resolved now.